README.md

```diff
@@ -53,7 +55,7 @@ You can find out more about Kani from the [Kani book](https://model-checking.git
 
 ## Contact
 
-For questions, suggestions or feedback, feel free to open an issue on the Kani page with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
+For questions, suggestions or feedback, feel free to open an [issue on the Kani page](https://github.com/model-checking/kani/issues) with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).
```

Comment:
Shouldn't they just create an issue in this repository instead?

Reply:
I thought we wanted to save issues in this repository for challenges, and we can use a discussions tab to drive discussions about these challenges, and meta-level issues with the contest itself should be directed to Kani. lmk if you disagree.